src/coverlet.core/Abstracts/IFileSystem.cs

```diff
@@ -1,7 +1,27 @@
-﻿namespace Coverlet.Core.Abstracts
+﻿using System.IO;
+
+namespace Coverlet.Core.Abstracts
 {
-    internal interface IFileSystem
+    public interface IFileSystem
     {
         bool Exists(string path);
+
+        void WriteAllText(string path, string contents);
+
+        string ReadAllText(string path);
+
+        Stream OpenRead(string path);
+
+        void Copy(string sourceFileName, string destFileName);
+
+        void Copy(string sourceFileName, string destFileName, bool overwrite);
+
+        void Delete(string path);
+
+        void AppendAllText(string path, string contents);
```

Comment:
why?Where is used?

Reply:
Sorry this was used in the ```coverlet.collector``` and I missed to revert it.